docs/topics/kapt.md

```diff
@@ -70,6 +70,28 @@ Follow these steps:
    `kaptAndroidTest` and `kaptTest`. Note that `kaptAndroidTest` and `kaptTest` extends `kapt`, so you can just provide the
    `kapt` dependency and it will be available both for production sources and tests.
 
+## Try Kotlin K2 compiler
+
+> Support for K2 in the kapt compiler plugin is [Experimental](components-stability.md).  Opt-in is required (see details below),
+> and you should use it only for evaluation purposes.
+>
+{type="warning"}
+
+From Kotlin 1.9.20, you can try using the kapt compiler plugin with the [K2 compiler](https://blog.jetbrains.com/kotlin/2021/10/the-road-to-the-k2-compiler/),
```

Comment:
My intention was that this documentation would only be available from the final release of 1.9.20, which is why I'm merging into `1-9-20-docs` branch. But if you think it would be better to make it available earlier by merging into `1-9-20-rc-eap-details`, please say and I will update the PR.

Reply:
Oh, sorry, my bad.
Need to have a rest :(